src/book/03-guides/04-general/03-glossary.mdx

```diff
@@ -174,6 +174,8 @@ slug: /guides/general/glossary
 
 [**Transformation**](<https://en.wikipedia.org/wiki/Transformation_(function)>): A function which maps a set to itself (sometimes called an operator on that set). Often, it is used in the context of a spatial manipulation - regularly representated as a translation and a rotation.
 
+**Units**: [S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used. Storage of values may have a different unit, e.g. [nbs and nbs index files](https://nubook.netlify.app/system/foundations/nuclear#nbs-files).
```

Comment:
```suggestion
**Units**: [S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](https://nubook.netlify.app/system/foundations/nuclear#nbs-files).
```

Reply:
Even time we generally store as std::chrono, which are unit aware (which is better than having a specific unit)

Reply:
The URL here needs to be updated to nubook.nubots.net